application/src/main/resources/db/V2__table_message_metadata.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE MESSAGE_METADATA(
+    MESSAGE_ID       BIGINT,
+    GUILD_ID         BIGINT,
+    CHANNEL_ID       BIGINT,
+    USER_ID          BIGINT,
+    CREATE_TIMESTAMP BIGINT
```

Comment:
While there's nothing wrong with using this, for your info, Snowflake ID's contain when it was created.
See TimeUtil#getTimeCreated(long) in JDA

So you honestly don't even need this

Reply:
We may need time travel queries in the database like `WHERE CREATE_TIMESTAMP > XXX`
A database doesn't understand snowflake hence its existence is justified.

Reply:
Not sure how the database exactly works, but I'd assume there's a way to look at the start of the ID
[Here](https://discord.com/developers/docs/reference#snowflakes) you can see how snowflake ID's are structured, start is timestamp

Reply:
In that case I forgive you, the guild ID is still useless though